output-googleassistant/README.md

```diff
@@ -1 +1,226 @@
+  listen: true,
+}
+```
+
+You can also add platform-specific output to an output template. [Learn more about Google Assistant-specific output below](#google-assistant-specific-output-elements).
+
+```typescript
+{
+  // ...
+  platforms: {
+    GoogleAssistant: {
+      // ...
+    }
+  }
+}
+```
+
+
+## Generic Output Elements
+
+Generic output elements are in the root of the output template and work across platforms. [Learn more in the Jovo Output docs](https://github.com/jovotech/jovo-output/blob/master/docs/output-templates.md).
+
+Below, you can find a list of generic output elements that work with Google Assistant.
+
+### Message
+
+The `message` is what Google Assistant is saying (or displaying) to the user:
+
+```typescript
+{
+  message: 'Hello world!',
+}
+```
+
+Under the hood, Jovo translates the `message` into a `firstSimple` response object ([see the official Google Assistant docs](https://developers.google.com/assistant/conversational/prompts-simple?hl=en)):
+
+```json
+{
+  "prompt": {
+    "firstSimple": {
+      "speech": "Hello world!"
+    }
+  }
+}
+```
+
+It's also possible to add a `displayText` property:
+
+```typescript
+{
+  message: {
+    text: 'Speech hello world!',
+    displayText: 'Display hello world!'
+  },
+}
+```
+
+This is then turned into the following response:
+
+```json
+{
+  "prompt": {
+    "firstSimple": {
+      "speech": "Speech hello world!",
+      "text": "Display hello world!"
+    }
+  }
+}
+```
+
+### Reprompt
+
+If Google Assistant asks a question and the user does not respond after a few seconds, it will state a `reprompt` to ask again:
+
+```typescript
+{
+  message: `Hello world! What's your name?`,
+  reprompt: 'Could you tell me your name?',
+  listen: true,
+}
+```
+
+Under the hood, Jovo translates the `reprompt` into `NO_INPUT_1`, `NO_INPUT_2`, and `NO_INPUT_FINAL`.
+
+**Note**: For Google Assistant to wait for a user to answer a question, the [listen property](#listen) needs to be added.
+
+### Listen
+
+The `listen` property needs to be added to tell Google Assistant that it should keep the microphone open and wait for a user's response.
+
+```typescript
+{
+  listen: true,
+}
+```
+
+If `listen` is not set to `true`, Jovo transitions to the `actions.scene.END_CONVERSATION` under the hood.
+
+### Quick Replies (Suggestions)
+
+On Google Assistant, quick replies are called suggestions. [You can learn more in the official Google Assistant docs](https://developers.google.com/assistant/conversational/prompts?hl=en#suggestions).
+
+```typescript
+{
+  // ...
+  quickReplies: [ 'Yes', 'No' ]
+}
+```
+
+### Card
+
+You can display a basic card ([see the official Google Assistant docs](https://developers.google.com/assistant/conversational/prompts-rich?hl=en)) by using the `card` property:
+
+```typescript
+{
+  // ...
+  card: {
+    title: 'Hello world!',
+    content: 'Welcome to this new app built with Jovo.'
+  },
+}
+```
+
+### Carousel (Collection Browse)
```

Comment:
Carousel Browse is not implemented due to being limited to web-links only, instead we use `Collection` which requires setting up a scene.